moto/awslambda/responses.py

```diff
@@ -300,6 +302,15 @@ def _delete_function(self, request, full_url, headers):
         else:
             return 404, {}, "{}"
 
+    @staticmethod
+    def _update_function_configuration(configuration, qualifier):
```

Comment:
The function name is a bit misleading, as it doesn't update - only copy and rename.

Reply:
what about `_set_configuration_qualifier` ?

Reply:
Sounds good!